src/toil/batchSystems/lsf.py

```diff
@@ -124,7 +124,7 @@ def getJobExitCode(self, lsfJobID):
                 return None
 
             # if not found in bjobs, then try bacct (slower than bjobs)
-            logger.debug("bjobs failed to detect job - trying bacct: "
+            logger.error("bjobs failed to detect job - trying bacct: "
```

Comment:
This should remain a `debug` level as we can still try `bacct`

Reply:
Yes sorry, this was a mistake :/